### PR TITLE
Pull request for #19

### PR DESCRIPTION
**Pull request by amihaiemil**
This PR is for issue #19 . I only added the javasdoc as the author said. @yegor256 said we should change the implementation of ``contains()`` so it has O(1) complexity, but I do not see how that is possible (see my comment from the issue) .

### CODE REVIEW

**Comment by amihaiemil** [comment]
The obenjdk6 fails with the same error as the master project, and it's not related to this PR. Harmcrest classes cannot be found in tests. Most likely it's because harmcrest needs jdk 7.

**Comment by dmarkov** [comment]
@amihaiemil I will find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by dmarkov** [comment]
@longtimeago it's yours, please review

File: src/main/java/com/jcabi/immutable/ArraySet.java [hunk 1]
```diff
@@ -187,6 +187,13 @@ public boolean isEmpty() {
         return this.values.length == 0;
     }
 
+    /**
+     * Checks if the given object is in this set or not.
+     * Since this set is implemented using an array,<br>
```
Comment (longtimeago) [inline src/main/java/com/jcabi/immutable/ArraySet.java]: @amihaiemil Better to write "this Set implementation is backed by array ..."

File: src/main/java/com/jcabi/immutable/ArraySet.java [hunk 2]
```diff
@@ -187,6 +187,13 @@ public boolean isEmpty() {
...
+     * this method works in linear time (<b>O(n)</b>).
```
Comment (longtimeago) [inline src/main/java/com/jcabi/immutable/ArraySet.java]: @amihaiemil complexity of the operation is `(<b>O(n)</b>)`

**Comment by longtimeago** [comment]
@amihaiemil I also don't see any other approach how to make overall complexity of the operation `O(1)`.
Maybe, Yegor has some ideas.

**Comment by longtimeago** [comment]
@amihaiemil please, fix the javadoc a bit according to my remarks. Thanks!

**Comment by amihaiemil** [comment]
@longtimeago done.

**Comment by longtimeago** [comment]
@rultor merge

**Comment by amihaiemil** [comment]
@yegor256 Please let rultor merge if you don't have other suggestions. Thanks.

**Comment by yegor256** [comment]
@rultor merge

**Comment by longtimeago** [comment]
@rultor deploy

**Comment by yegor256** [comment]
@rultor deploy

**Comment by dmarkov** [comment]
@longtimeago thanks, paid, **21 mins** to your account, payment ID is `77676964`, task took 118 hours and 9 mins to complete. extra minutes for review comments (c=6). added +21 to your rating, now it is equal to [+4669](http://www.netbout.com/b/35024?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy pls

**Pull request merged**